xray/fiber.go

```diff
@@ -0,0 +1,61 @@
+package xray
+
+import (
+	"context"
+	"github.com/aws/aws-xray-sdk-go/header"
+	"github.com/gofiber/fiber/v2"
+)
+
+type FiberHandler interface {
+	Handler(SegmentNamer, fiber.Handler) fiber.Handler
+}
+
+type fiberHandler struct {
+	cfg *Config
+}
+
+// NewFiberInstrumentor returns a new FiberHandler that
+// provides a Handler function to satisfy the fiber.Handler
+// interface.
+func NewFiberInstrumentor(cfg *Config) FiberHandler {
+	return &fiberHandler{
+		cfg: cfg,
+	}
+}
+
+// Handler wraps the provided fiber.Handler.
+func (h *fiberHandler) Handler(sn SegmentNamer, handler fiber.Handler) fiber.Handler {
```

Comment:
I think probably a good idea to provide context as an argument of this method to provide abilities to cancel the context or pass the context with a deadline.